### ISSUE TITLE
Standard well-known error classes for common things

### ISSUE THREAD

**Issue by jcbhmr** [jcbhmr/jcbhmr#33]
### Discussed in https://github.com/jcbhmr/jcbhmr/discussions/6

<div type='discussions-op-text'>

<sup>Originally posted by **jcbhmr** June 25, 2022</sup>
A nice standard simplistic NPM package or other URL import would be nice for things like...
- AssertionError
- HttpError (400, 401, etc.)
- NotImplementedError
- UnreachableError extends LogicError
- FormatError extends RangeError (RangeError is already used for non-conforming strings by some DOM APIs)
- LogicError

Basically, I'm looking for these classes a lot (maybe not _a lot_ but often enough that I Google for "npm unreachable error" enough to make it autocomplete) and I want a search result to pop up and say "here it is! use this one! its used by other people too, so you can `instanceof` check it!</div>
